tools/nav2_bashrc

```diff
@@ -0,0 +1,14 @@
+OVERLAY_DIR=/opt/overlay_ws
+
+# https://docs.ros.org/en/rolling/Tutorials/Workspace/Creating-A-Workspace.html#source-the-overlay
+OVERLAY=$OVERLAY_DIR/install/setup.bash
+UNDERLAY=/opt/ros/${ROS_DISTRO}/setup.bash
+if [ -f "$OVERLAY" ]; then
+    source $OVERLAY
+else
+    source $UNDERLAY
+fi
+
+# https://code.visualstudio.com/remote/advancedcontainers/persist-bash-history
```

Comment:
I see the vscode docs recommends to bake this into the Dockerfile, but I think that's just adds unnecessary opportunities to bust the docker layer cache when it doesn't have to. Please add such user session config to the `postCreateCommand` where we already manage things like .bashrc.

https://github.com/ros-navigation/navigation2/blob/d302f1ad53e0af1d55ddc94214ade7e5f9010f44/.devcontainer/post-create-command.sh#L10-L11  

- https://containers.dev/implementors/spec/#post-container-creation